src/components/Navigation.js

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
 import { NavLink } from 'react-router-dom';
+import React,{useState, useContext} from 'react';
```

Comment:
Because we don't `useState()` inside of this file, we can remove it from the imports '' '